phoenicis-configuration/src/main/java/org/phoenicis/configuration/localisation/Localisation.java

```diff
@@ -50,28 +50,37 @@ public static void setAdditionalTranslations(ResourceBundle resourceBundle) {
         getI18n().setResources(resourceBundle);
     }
 
-    public static String tr(String str) {
-        return getI18n().tr(str);
+    public static String tr(String text) {
+        return text == null ? "" : getI18n().tr(text);
     }
 
     public static String tr(String text, Object o1) {
-        return getI18n().tr(text, o1);
+        return text == null ? "" : getI18n().tr(text, o1);
     }
 
     public static String tr(String text, Object o1, Object o2) {
-        return getI18n().tr(text, o1, o2);
+        return text == null ? "" : getI18n().tr(text, o1, o2);
     }
 
     public static String tr(String text, Object o1, Object o2, Object o3) {
-        return getI18n().tr(text, o1, o2, o3);
+        return text == null ? "" : getI18n().tr(text, o1, o2, o3);
     }
 
     public static String tr(String text, Object o1, Object o2, Object o3, Object o4) {
-        return getI18n().tr(text, o1, o2, o3, o4);
+        return text == null ? "" : getI18n().tr(text, o1, o2, o3, o4);
     }
 
     public static String tr(String text, Object[] objects) {
-        return getI18n().tr(text, objects);
+        return text == null ? "" : getI18n().tr(text, objects);
+    }
+
+    /**
+     * Translates the given Translatable object, by creating a new instance of it with its corresponding translations
+     * @param translatable Translatable object
+     * @return translated object
+     */
+    public static Translatable tr(Translatable translatable) {
```

Comment:
I think it would be better to provide a template method and generic interface for `Translatable` here:
```java
public static <T> Translatable<T> tr(Translatable<T> translatable) {
   return translatable.translate();
}
```
This would remove the need to cast the resulting element later on.